ksops.go

```diff
@@ -13,14 +13,54 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 
+	"github.com/joho/godotenv"
 	"go.mozilla.org/sops/v3/cmd/sops/formats"
 	"go.mozilla.org/sops/v3/decrypt"
 	"sigs.k8s.io/yaml"
 )
 
+type Metadata struct {
```

Comment:
Instead of redefining our own struct here, can we use [types.ObjectMeta](https://github.com/kubernetes-sigs/kustomize/blob/master/api/types/objectmeta.go#L8)? Or at least reference it in a comment

Reply:
Switched to [types.ObjectMeta](https://github.com/kubernetes-sigs/kustomize/blob/master/api/types/objectmeta.go#L8)